webpack.dev.config.js

```diff
@@ -0,0 +1,38 @@
+const { createConfig } = require('@edx/frontend-build');
+
+const path = require('path');
+const fs = require('fs');
+const webpack = require('webpack');
+
+const basePluginsPath = path.resolve(__dirname, 'plugins/');
+
+const pluginTabs = fs.readdirSync(basePluginsPath).map(
+    dir => {
+        console.log(path.resolve(basePluginsPath, dir, 'package.json'));
+        const meta = require(path.resolve(basePluginsPath, dir, 'package.json'));
+        console.log(meta['open-edx-meta'].tabs);
+        return {
+            id: dir,
+            ...meta['open-edx-meta'].tabs,
+        };
+
+    }
+)
```

Comment:
This is going through each directory in ./plugins, reading the package.json file and getting metadata from it. Instead of package.json we can use our own system.